test/asar.js

```diff
@@ -53,3 +54,40 @@ util.testSinglePlatform('asar test', (t, opts) => {
       return fs.stat(path.join(resourcesPath, 'app.asar.unpacked', 'dir_to_unpack'))
     }).then(stats => t.true(stats.isDirectory(), 'dir_to_unpack should exist under app.asar.unpacked subdirectory when opts.asar-unpack-dir is set dir_to_unpack'))
 })
+
+util.testSinglePlatform('prebuilt asar test', (t, opts) => {
+  opts.name = 'prebuiltAsarTest'
+  opts.dir = util.fixtureSubdir('asar-prebuilt')
+  opts.asar = {
+    'unpack': '*.pac',
+    'unpackDir': 'dir_to_unpack',
+    'filename': path.join(opts.dir, 'app.asar')
+  }
+  let resourcesPath
+  sinon.spy(console, 'warn')
+  return util.packageAndEnsureResourcesPath(t, opts)
+    .then(generatedResourcesPath => {
+      console.warn.calledWithExactly('WARNING: asar.filename has been specified, all other asar options will be ignored')
```

Comment:
Shouldn't this be wrapped in `t.true()`?

Reply:
sorry, i’ve been in jest land for a while, i assumed sinon made the assertion.  i’ll update.

Reply:
done